code/modules/mob/living/simple_animal/hostile/pulse_demon/pulsedemon.dm

```diff
@@ -152,7 +182,9 @@
 	..()
 	var/turf/T = get_turf(src)
 	spark(src,rand(2,4))
-	empulse(T,charge/50000,charge/25000)
+	var/heavyemp_radius = min(charge/50000, 20)
+	var/lightemp_radius = min(charge/25000, 25)
+	empulse(T, heavyemp_radius, lightemp_radius)
```

Comment:
This is fine but it wasn't documented in the OP

Reply:
the server got into an infinite loop and i had to kill it with taskmgr without this if i died with a ton of charge

Reply:
I'm not saying it's wrong just saying you should make sure to do your best to include any substantial bugfixes (e.g.: infinite loops) or balance changes (e.g.: capping the EMP). I think the change itself is completely reasonable, and 25 is perhaps even more generous than necessary but fine enough.